frontend/components/analysis/StatusChip.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { type Analysis } from '@/openapi/generated/pacta'
+
+const { t } = useI18n()
+const prefix = 'components/analysis/StatusChip'
+const tt = (s: string) => t(`${prefix}.${s}`)
+
+interface Props {
+  analysis: Analysis
+}
+const props = defineProps<Props>()
+
+const isStale = (a: Analysis): boolean => new Date().getTime() - new Date(a.createdAt).getTime() > 1000 * 60 * 10
```

Comment:
Seems like there's a lot of overlapping logic between here and `ContextualListView`, maybe shove that into a `lib` somewhere?

Reply:
Great call - that was part of my goal of moving this to `StatusChip`, so I actually can fully remove it from the ContextualListView.